bigquery/indexer.py

```diff
@@ -202,12 +204,13 @@ def _docs_by_id_from_export(storage_client, bucket_name, export_obj_prefix,
 
 def _tsv_scripts_by_id_from_export(storage_client, bucket_name,
                                    export_obj_prefix, table_name,
-                                   participant_id_column, time_series_column):
+                                   participant_id_column, time_series_column,
+                                   time_series_type):
     for row in _rows_from_export(storage_client, bucket_name,
                                  export_obj_prefix):
         participant_id = row[participant_id_column]
         del row[participant_id_column]
-        tsv = row[time_series_column]
+        tsv = str(time_series_type(row[time_series_column])).replace('.', '_')
```

Comment:
```
# Say time series value is 4.5. If the field name ended with "4.5", then when we lookup this field in Elasticsearch, Elasticsearch thinks we are looking for a field "5" instead a nested object named "4".
# Use _ instead of . to avoid this confusion.
```

Reply:
I've tested these changes on a table that had time values of 0 and 0.5 (one of the only tables I could find without null values and with non-integer values).